vendor/bat-native-ledger/src/bat/ledger/internal/legacy/media/vimeo.cc

```diff
@@ -190,54 +187,32 @@ bool Vimeo::IsExcludedPath(const std::string& path) {
     return true;
   }
 
-  const std::vector<std::string> paths({
-      "/",
-      "/log_in",
-      "/upgrade",
-      "/live",
-      "/watch",
-      "/videoschool",
-      "/upload",
-      "/ondemand",
-      "/ott",
-      "/site_map",
-      "/blog",
-      "/help",
-      "/about",
-      "/jobs",
-      "/stats",
-      "/watchlater",
-      "/purchases",
-      "/settings",
-      "/stock",
-    });
-
-  for (std::string str_path : paths) {
-    if (str_path == path || str_path + "/" == path) {
+  static constexpr std::array<const char*, 19> kPaths{
+      "/",          "/log_in",      "/upgrade", "/live",
+      "/watch",     "/videoschool", "/upload",  "/ondemand",
+      "/ott",       "/site_map",    "/blog",    "/help",
+      "/about",     "/jobs",        "/stats",   "/watchlater",
+      "/purchases", "/settings",    "/stock",
+  };
+
+  for (const auto* kPath : kPaths) {
```

Comment:
I'm thinking that the `k` prefix should not be used for a local var? Maybe `excluded_path` or similar?

Reply:
The use of `k` prefix here is a requirement from the google style guide, and I think it is appropriate since this is not a regular local const variable, but a static constexpr.

https://google.github.io/styleguide/cppguide.html#Constant_Names

Reply:
Fixed.